### PR TITLE
Reader: run prettier on Reader blocks

### PR DESCRIPTION
**Pull request by bluefuton**
Run wp-prettier on Reader-specific blocks in `client/blocks`.

No functional changes.

### CODE REVIEW

**Comment by bluefuton** [comment]
The lint errors causing the build to fail are all related to CSS naming, and can be safely ignored for the moment.

File: client/blocks/reader-combined-card/index.jsx [hunk 1]
```diff
@@ -59,18 +60,27 @@ class ReaderCombinedCard extends React.Component {
...
+		} = this.props;
 		const feedId = postKey.feedId;
 		const siteId = postKey.blogId;
 		const siteIcon = get( site, 'icon.img' );
 		const feedIcon = get( feed, 'image' );
 		const streamUrl = getStreamUrl( feedId, siteId );
 		const siteName = getSiteName( { site, post: posts[ 0 ] } );
 		const isSelectedPost = post => keysAreEqual( keyForPost( post ), selectedPostKey );
-		const followUrl = feed && feed.URL || site && site.URL;
+		const followUrl = ( feed && feed.URL ) || ( site && site.URL );
```
Comment (samouri) [inline client/blocks/reader-combined-card/index.jsx]: so cool

File: client/blocks/reader-share/index.jsx [hunk 2]
```diff
@@ -183,22 +182,27 @@ class ReaderShare extends React.Component {
...
+			'is-active': this.state.showingMenu,
 		} );
 
-		return React.createElement( this.props.tagName, {
-			className: 'reader-share',
-			onClick: this.toggle,
-			onTouchStart: this.preloadEditor,
-			onMouseEnter: this.preloadEditor,
-			ref: 'shareButton' },
+		return React.createElement(
```
Comment (samouri) [inline client/blocks/reader-share/index.jsx]: always funny to spot a React.createElement!

**Comment by samouri** [comment]
Code LGTM.  Testing now

@bluefuton, what was the command you ran? 
I plan on hitting all these files with the createClass deprecation codeshift after this

**Comment by bluefuton** [comment]
@samouri I got most of them with:

```prettier --write reader-*/**/*.js*
prettier --write reader-*/*.js*```

...then did the same manually for a few blocks without the `reader-` prefix.

Yay for the createClass codemod.

File: client/blocks/reader-combined-card/index.jsx (hunk 1 above)
Comment (blowery) [inline client/blocks/reader-combined-card/index.jsx]: also, it's wrong. :) 

Should be looking for feed_URL on the site first. Switch to using the helper?

**Pull request merged**